packages/cli-core/src/loggers/amplify_event_loggers.ts

```diff
@@ -191,6 +220,26 @@ export class AmplifyEventLogger {
             msg.data.duration / 1000
           } seconds`,
         );
+        const synthesisTime = this.synthesisDuration ?? 0;
+        const typeCheckTime = this.typeCheckDuration ?? 0;
+        const deploymentTime = msg.data.duration;
+        const latencyDetails: LatencyDetails = {
+          total: synthesisTime + typeCheckTime + deploymentTime,
+          synthesis: synthesisTime,
+        };
+        if (this.isHotswapDeployment) {
+          latencyDetails.hotSwap = deploymentTime;
+        } else {
+          latencyDetails.deployment = deploymentTime;
+        }
+        await this.telemetryDataEmitter.emitSuccess(latencyDetails, {
+          subCommands: 'SandboxEvent',
```

Comment:
```suggestion
          subCommands: 'SandboxDeployment',
```

maybe that's better name for this event ?


additionally - is this the right place to do this? 
should we have sandbox -> event logger -> metrics
or fan out
sandbox -> event logger
sandbox -> metrics

I would say that event logging and metrics are two separate concerns that shouldn't be coupled together.